app/server-utils/stampy.ts

```diff
@@ -283,13 +283,15 @@ export const loadTag = withCache('tag', async (tagName: string): Promise<Tag> =>
 
   const questions = await loadAllQuestions()
   const nameToId = Object.fromEntries(
-    questions.data.filter((q) => q.status == QuestionStatus.LIVE).map((q) => [q.title, q.pageid])
+    questions.data
+      .filter((q) => q.status == QuestionStatus.LIVE_ON_SITE)
```

Comment:
I wonder if there should be a general filter function to handle this? I'm guessing there will be more such situations in the future :/

Reply:
sure, but it sounded like "a future problem" to me right now 😅